src/containers/SCButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Again, you're mixing component (markup) with container (data flow).

Perhaps renaming this code to `component/SCButton` and `container/TrackLoader` will help. Then write a new component called `component/SimpleButton`. It should be EASY to swap between two "dumb display things that call a passed `getTracks` method" underneath the redux container `TrackLoader`.

You should be able to swap `SCButton` and `SimpleButton` out with an `import` change and a different argument to HOC creator `connect()` thingy.